src/main/java/se/sundsvall/messaging/service/HistoryService.java

```diff
@@ -35,4 +38,13 @@ public List<History> getConversationHistory(final String municipalityId, final S
 		return dbIntegration.getHistory(municipalityId, partyId, from, to);
 	}
 
+	public UserMessages getUserMessages(final UserMessagesRequest request, final String municipalityId) {
```

Comment:
getUserStatistics?

Reply:
Technically not statistics, the requirements says they want to see each and every message with the ability to also fetch attachments. 

Statistics would be "this many messages were sent", "this portion of the sent messages were sent with digital mail" etc etc.